api/api.go

```diff
@@ -211,10 +211,11 @@ func (s *SignRequest) Validate() error {
 
 // SignResponse is the response object of the certificate signature request.
 type SignResponse struct {
-	ServerPEM  Certificate          `json:"crt"`
-	CaPEM      Certificate          `json:"ca"`
-	TLSOptions *tlsutil.TLSOptions  `json:"tlsOptions,omitempty"`
-	TLS        *tls.ConnectionState `json:"-"`
+	ServerPEM    Certificate          `json:"crt"`
+	CaPEM        Certificate          `json:"ca"`
+	CertChainPEM []Certificate        `json:"certChain"`
```

Comment:
`CaPEM` used to be the issuer certificate (intermediate) and it was used to construct the bundle joining both `ServerPEM` and `CaPEM`, now to construct the bundle we will need to bundle `ServerPEM` with the full `CertChainPEM`. Keeping both should not be necessary, and we should consolidate to one.

Reply:
From my point view I want to deprecate/remove `ServerPEM` and `CaPEM` and just leave only `CertChainPEM`. What do you think?

Reply:
It makes sense to me.

Reply:
But before making the changes, let me discuss it internally, I know other projects also split the server certificates and intermediates.

Reply:
Any news?

Reply:
Hey @jkralik apologies for the radio silence. We've been going back and forth on how we felt about changing the API (in general we're all for returning the chain). 

This kinda got lost in a jumble between a few other things we've got going on, so we honestly appreciate the reminder. I like the names you've chosen (we argued about names for a while; we like to argue) and I'm giving the PR a last once-over at the moment with the intention of merging / requesting any small changes in a bit.